presto-plugin-toolkit/src/main/java/io/prestosql/plugin/base/security/FileBasedAccessControl.java

```diff
@@ -318,6 +319,12 @@ public void checkCanShowRoleGrants(ConnectorSecurityContext context, String cata
     {
     }
 
+    @Override
+    public void checkCanExecuteProcedure(ConnectorSecurityContext context, SchemaProcedureName procedure)
+    {
+        //TODO implement
```

Comment:
Instead please create an issue if you think this should be implemented, or just implement it.

Reply:
i asked for this & asked for a follow-up too